tsconfig.json

```diff
@@ -27,9 +27,9 @@
         // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
         /* Modules */
-        "module": "nodenext" /* Specify what module code is generated. */,
+        "module": "NodeNext" /* Specify what module code is generated. */,
         // "rootDir": "./",                                  /* Specify the root folder within your source files. */
-        "moduleResolution": "nodenext" /* Specify how TypeScript looks up a file from a given module specifier. */,
+        "moduleResolution": "NodeNext" /* Specify how TypeScript looks up a file from a given module specifier. */,
```

Comment:
### Note to reviewer

this is to resolve a warning from VSCode

![image](https://github.com/jessealama/decimal128/assets/8733840/7e116fd5-fb32-43e2-92bf-47d788b51ac2)

Reply:
Oops -- thanks!